ios/PresentationLayer/UIToolkit/Sources/UIToolkit/Extensions/View+Extensions.swift

```diff
@@ -31,3 +31,9 @@ public extension View {
             .shimmering(active: condition(), duration: duration, bounce: bounce)
     }
 }
+
+public extension View {
+    func toastView(toastData: Binding<ToastData?>) -> some View {
```

Comment:
Maybe change the param to `_ toastData` so it would be easier to use? 🙂

Reply:
Done